spec/index.html

```diff
@@ -4841,9 +4830,7 @@ <h4>Effective Boolean Value (EBV)</h4>
             <li>If the argument is a <span class="type typedLiteral">typed literal</span> with a
               <span class="type datatype">datatype</span> of <code>xsd:boolean</code>, and it has a
               valid lexical form, the EBV is the value of that argument.</li>
-            <li>If the argument is a <span class="type plainLiteral">plain literal</span> or a
-              <span class="type typedLiteral">typed literal</span> with a <span class=
-                                                                                "type datatype">datatype</span> of <code>xsd:string</code>, the EBV is false if the
+            <li>If the argument is a <span class="type simpleLiteral">simple literal</span>, the EBV is false if the
```

Comment:
Since simple literals are something that exists only in concrete syntaxes, I would say that they are not relevant here. So, this part about the simple literal should be removed.

However, removing the case of typed literals with datatype xsd:string is incorrect.

Hence, this line should be changed as follows.
```suggestion
            <li>If the argument is a <span class="type typedLiteral">typed literal</span> with a
              <span class="type datatype">datatype</span> of <code>xsd:string</code>, the EBV is false if the
```